moses/vae/model.py

```diff
@@ -127,7 +127,7 @@ def forward_decoder(self, x, z):
 
         z_0 = z.unsqueeze(1).repeat(1, x_emb.size(1), 1)
         x_input = torch.cat([x_emb, z_0], dim=-1)
-        x_input = nn.utils.rnn.pack_padded_sequence(x_input, lengths,
```

Comment:
.cpu() not needed, here lengths is a list of integers (living on the CPU altready). Also this will probably fail, as list does not. have a .cpu() method

Reply:
I have just pushed this change in the latest commit :)